### ISSUE TITLE
Deleting a User with a download causes a PHP Warning

### ISSUE THREAD

**Issue by eSilverStrike** [Geeklog-Plugins/downloads#23]
Was batch deleting a user in the Admin and the following error happened

```
A PHP error has occurred:

E_WARNING(2) - Undefined array key "delete_downloads" @ /usr/home/geeklog/geeklog/plugins/downloads/functions.inc line 789

Call Stack
--

1 | plugin_user_delete_downloads | /usr/home/geeklog/geeklog/system/lib-plugins.php | 1395
2 | PLG_deleteUser | /usr/home/geeklog/geeklog/system/lib-user.php | 91
3 | USER_deleteAccount | /usr/home/geeklog/public_html/admin/user.php | 1197
4 | batchdeleteexec | /usr/home/geeklog/public_html/admin/user.php | 1567
```

Looks like $_DLM_CONF['delete_downloads'] is not set even though the config option exists in the database. the variable $_DLM_CONF is set to global in the  plugin_user_delete_downloads function.

This needs to be looked into further.

**Also need to add a plugin_usercontributed_downloads function** so any user downloads are listed (as I wouldn't have tried to delete this user had I know. Comments and Articles use this function

**Comment by eSilverStrike** [comment]
@mystralkk  I added theplugin_usercontributed_downloads  function with commit 9431cd7eb6557bf54d330e27e0dfa6f72f31143a but the other issue I can't fix due to not having access to my development server over the next 2 months. If you get a chance can you look at it?

**Comment by mystralkk** [comment]
Fixed with change set  8c74153.   The right property name was not $_DLM_CONF['delete_downloads'] but $_DLM_CONF['delete_download'].
